src/components/Error/Error.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './Error.scss'
+import {error} from '../../assets/images/svg/Exclamation_Circle_Red.svg'
+
+class Error extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            errorMeg: this.props.errorMeg
+        };
+      }
+      
+      render() {
+        return <span>
+        <svg src={error} />
```

Comment:
`src` doesn't work with `<svg>` elements.  Change this back to `<img>` and it will be fine.